### DISCUSSION TITLE
Prebuilt dev containers - for the hooks included in the image?

### DISCUSSION THREAD

**Discussion by diberry** [devcontainers/community#108] [Q&A] [answered]
If I use a postCreateCommand or any of the hooks, are those actions included in the prebuilt image? How can I change me workload to make sure those actions are in the prebuilt image?

**Comment by diberry** [comment, marked as answer]
Found this video - https://www.youtube.com/watch?v=M21loGvplVM

**Reply by samruddhikhandale** [reply]
Awesome! We also have a guide, see https://containers.dev/guide/prebuild
